{
  "Header": {
    "Original": "CVE-2020-8648 (There is a use-after-free vulnerability in the Linux kernel through 5. ...)",
    "Line": 248985,
    "ID": "CVE-2020-8648",
    "Description": "(There is a use-after-free vulnerability in the Linux kernel through 5. ...)"
  },
  "Annotations": [
    {
      "Original": "{DSA-4698-1 DLA-2242-1 DLA-2241-1}",
      "Line": 248986,
      "Type": "xref",
      "Bugs": [
        "DSA-4698-1",
        "DLA-2242-1",
        "DLA-2241-1"
      ]
    },
    {
      "Original": "- linux 5.5.13-1",
      "Line": 248987,
      "Type": "package",
      "Package": "linux",
      "Kind": "fixed",
      "Version": "5.5.13-1"
    },
    {
      "Original": "[buster] - linux 4.19.118-1",
      "Line": 248988,
      "Type": "package",
      "Release": "buster",
      "Package": "linux",
      "Kind": "fixed",
      "Version": "4.19.118-1"
    },
    {
      "Original": "NOTE: https://git.kernel.org/linus/07e6124a1a46b4b5a9b3cacc0c306b50da87abf5",
      "Line": 248989,
      "Type": "NOTE",
      "Description": "https://git.kernel.org/linus/07e6124a1a46b4b5a9b3cacc0c306b50da87abf5"
    }
  ]
}